wafer/users/templates/wafer.users/profile.html

```diff
@@ -110,6 +110,11 @@
 {% for talk in profile.pending_talks %}
 <div class="well">
     <a href="{{ talk.get_absolute_url }}">{{ talk.title }}</a>
+    <!-- Because this is one of the author's pending talks, we don't need to
+         check for edit permission's on the talk explictly. This doesn't show
+         the edit button for people with 'change-talk' permissions, but we
+         accept that tradeoff for simplicity here. -->
```

Comment:
This comment will show up in the generated HTML, which I'm guessing is not what we want? Should we use `{# ... #}` or `{% comment ... %} ... {% endcomment %}` instead?